dozer/cogs/ftc.py

```diff
@@ -232,12 +232,12 @@ def __init__(self, bot: commands.Bot):
         self.scparser = ScoutParser(self.http_session)
 
     @group(invoke_without_command=True, aliases=["ftcteam", "toa", "toateam", "ftcteaminfo"])
-    async def ftc(self, ctx: DozerContext, team_num: int):
+    async def ftc(self, ctx: DozerContext, *, team_name: str):
```

Comment:
I would prefer not to change the existing behavior, but adding the new command would be great!